solidity/contracts/bridge/Bridge.sol

```diff
@@ -226,6 +229,7 @@ contract Bridge is Governable, EcdsaWalletOwner {
         self.redemptionTimeout = 172800; // 48 hours
         self.movingFundsTxMaxTotalFee = 10000; // 10000 satoshi
         self.movingFundsTimeout = 7 days;
+        self.movingFundsDustThreshold = 1000; // 1000 satoshi
```

Comment:
This value should probably be higher than `movingFundsTxMaxTotalFee` since that parameter is the highest fee we think may be necessary to pay for the moving funds transaction.

Reply:
Makes sense! See https://github.com/keep-network/tbtc-v2/pull/232/commits/7836dbd33df102464250d4d4ff0eb1addd506609